src/components/SessionCard/SessionCard.tsx

```diff
@@ -16,80 +17,92 @@ const weekday = [
   "Saturday",
 ];
 
-function checkMark(status: SessionCardStatus) {
+function CheckMark({ status }: { status: SessionCardStatus }) {
```

Comment:
What... Thanks for the refactor. Can't believe this went through. Should do better reviews...